.circleci/config.yml

```diff
@@ -28,7 +28,6 @@ jobs:
       - *restore_cache
       - run: |
           npm ci
-          cd node_modules/mattermost-redux && npm i && npm run build
```

Comment:
Can you please elaborate why we don't want to cache the `node_modules` from redux any longer?

Reply:
This step is no longer needed for building webapp. Not related to cache